examples/SHT3X/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# The following lines of boilerplate have to be in your project's
+# CMakeLists in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.5)
+
+set(EXTRA_COMPONENT_DIRS "../../../../../hulp")
```

Comment:
Should be "../../../hulp" in this case, but I think this example might even be better located at eg. examples/I2C/Bitbang/Cmd/SHT3X ?